commands/deploy.go

```diff
@@ -268,7 +274,14 @@ func addFileToTarWriter(filePath string, tarWriter *tar.Writer, prefix string) e
 	// must provide real name
 	// (see https://golang.org/src/archive/tar/common.go?#L626)
 	header.Name = filepath.ToSlash(baseFilePath)
-
+  // ensure windows provides filemodes for binaries in node_modules/.bin
+	if runtime.GOOS == "windows" {
+		match := strings.Contains(baseFilePath,"node_modules\\.bin")
+		if match == true {
+			header.Mode = 511 // This is the value that it spat out when i ran npm i on linux then checked the int64 value of the stat.Mode() on files in node_modules/.bin/
```

Comment:
Why not octal literals? 511 decimal would be `0o777`. That said, `0o755` would be a better mode to use.
https://golang.org/ref/spec#Integer_literals

Reply:
I set it to what npm i set the filemode to be.

I agree 493 would be better, but i'm not quite sure who the owner is when the tarball is extracted.

Reply:
@jstangroome how did you know that this int64 value is an octal literal?

strange encoding if you ask me. I just reverse engineered via strconv/printf

Reply:
File mode is traditionally octal. 🤷